.gitignore

```diff
@@ -4,8 +4,8 @@ target
 .classpath
 .project
 /.idea
-/.gradle
-/build
+**/.gradle
+**/build
```

Comment:
These two changes are redundant since we are coming back to the single module project again.
Will fix on merge.